zmdocument.cls

```diff
@@ -191,3 +191,12 @@
 \renewcommand{\arraystretch}{1.5}
 
 \newcommand{\horizontalLine}{{\color{gray}\rule{\textwidth}{0.2pt}}}
+
+%%% Abbreviation
+
+\newcommand{\abbr}[2]{\dotuline{\gls{#1}}\newglossaryentry{#1}{
+  name=#1,
+  description={#2}
+}}
+
+\AtEndDocument{\printglossary[title=Liste des abbréviations]}
```

Comment:
To avoid any issues, should go in [the other AtEndDocument](https://github.com/zestedesavoir/latex-template/blob/master/zmdocument.cls#L184) :)